### PR TITLE
Updating Major Data Request

### PR DESCRIPTION
**Pull request by imanmoreira**
Updating endpoint call for searchneu's staging environment

### CODE REVIEW

File: common/constants.ts [hunk 1]
```diff
@@ -0,0 +1,95 @@
+// majors on which the scraper has been verified to run correctly:
+export const supported2018_2019 = [
```
Comment (imanmoreira) [inline common/constants.ts:2]: Moved these from scraper tests to common so they can be used for the mapping of supported majors

**Comment by imanmoreira** [comment]
[Update graphQL call](https://trello.com/c/8agCY5kW/281-update-graphql-call)

File: frontend/src/utils/fetchMajorsAndPlans.ts [hunk 2]
```diff
@@ -74,8 +73,12 @@ export function fetchMajorsAndPlans() {
           if (res.error) {
             throw res.error;
           }
+          console.log("reached done");
```
Comment (ArunJ98) [inline frontend/src/utils/fetchMajorsAndPlans.ts:76]: do we need these console logs?

File: frontend/src/utils/plan-helpers.ts [hunk 3]
```diff
@@ -14,6 +14,7 @@ export function findMajorFromName(
   if (!major) {
     return undefined;
   }
+  console.log(major.yearVersion);
```
Comment (ArunJ98) [inline frontend/src/utils/plan-helpers.ts:17]: same as above

File: frontend/src/utils/fetchMajorsAndPlans.ts (hunk 2 above)
Comment (imanmoreira) [reply frontend/src/utils/fetchMajorsAndPlans.ts:76]: LOL thanks

**Review by ArunJ98 — APPROVED** [review]

**Pull request merged**